firestore/examples/end2end/src/package.json

```diff
@@ -11,7 +11,7 @@
     "fs": "0.0.1-security",
     "google-auth-library": "^0.12.0",
     "google-protobuf": "^3.0.0",
-    "googleapis": "^23.0.0",
+    "googleapis": ">=39.1.0",
```

Comment:
This is frowned upon in node.js.  Semver major releases generally have breaking changes in them, so blindly accepting any major version will likely break things here.  I would recommend checking out why this file wasn't getting automatically scanned by renovatebot, and letting it submit these kinds of PRs.  The `google-auth-library` version used here is also astonishingly old.

Reply:
I see. Got it. Will go that direction. Thanks for the tips!